build-tools/mono-runtimes/mono-runtimes.projitems

```diff
@@ -15,20 +15,7 @@
   </PropertyGroup>
   <ItemGroup>
     <_MonoRuntime Include="armeabi" Condition=" '$(_ArmeabiRuntimeConfigure)' == 'true' ">
-      <Ar>$(_ArmAr)</Ar>
-      <As>$(_ArmAs)</As>
-      <Cc>$(_ArmCc)</Cc>
-      <Cpp>$(_ArmCpp) $(_ArmCppFlags)</Cpp>
-      <CFlags>$(_ArmCFlags) -march=armv5te $(_TargetCFlags)</CFlags>
-      <Cxx>$(_ArmCxx)</Cxx>
-      <CxxFlags>$(_ArmCxxFlags) -march=armv5te $(_TargetCxxFlags)</CxxFlags>
-      <CxxCpp>$(_ArmCxxCpp) $(_ArmCppFlags)</CxxCpp>
-      <Ld>$(_ArmLd)</Ld>
-      <LdFlags>$(_ArmLdFlags)</LdFlags>
-      <Objdump>$(_ArmObjdump)</Objdump>
-      <RanLib>$(_ArmRanLib)</RanLib>
-      <Strip>$(_ArmStrip)</Strip>
-      <ConfigureFlags>--host=armv5-linux-androideabi $(_TargetConfigureFlags)</ConfigureFlags>
+      <MonoSdks>targetarmv5</MonoSdks>
```

Comment:
just a nit, but would it be possible to change this to `target-armv5` (and all the other targets accordingly)? It just makes it easier to read and scan through the file

Reply:
Related nit: can we please continue using our existing ABI names of `armeabi` ("armv5"), `armeabi-v7a` ("armv7"), `arm64-v8a` (arm64), `x86`, and `x86_64`?

Consistency is nice, and those are the ABI directory names used within `.apk`s.

Reply:
Also, the lack of any separator between "target" and ABI is unreadable, IMHO. :-D

Reply:
I will rename to whatever is best for you 😄 . I prefixed it with `target` as it's the way we did it for iOS, but I am definitely ok with changing that to `armeabi`, `armeabi-v7a`, etc.

Reply:
I wasn't so much saying "get rid of `target`" as much as "use our existing ABI names within the target name", which would mean `target-armeabi`/`target-armeabi-v7a`/etc., *not* `targetarmv5`/`targetarmv7`/etc.